app/code/Magento/QuoteAnalytics/README.md

```diff
@@ -1,3 +1,17 @@
-# Magento_QuoteAnalytics
+# Magento_QuoteAnalytics module
 
-The Magento_QuoteAnalytics module configures data definitions for a data collection related to the Quote module entities to be used in [Advanced Reporting](https://devdocs.magento.com/guides/v2.4/advanced-reporting/modules.html).
+This module configures data definitions for a data collection related to the Quote module entities to be used in [Advanced Reporting](https://devdocs.magento.com/guides/v2.3/advanced-reporting/modules.html).
+
+## Installation
```

Comment:
Please add a mention that this module doesn't introduce any db modifications or new data (so everyone reading the readme knows that this is what happens - a lack of any mention might be considered just an omission, not as info that nothing gets added).